### ISSUE TITLE
nuclio function error

### ISSUE THREAD

**Issue by chang50961471** [opencv/cvat#7195]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)

### Steps to Reproduce

![微信图片_20231127153214](https://github.com/opencv/cvat/assets/31345352/7c7541d0-a66c-4cd5-88a3-ceb05a9c8d09)
![微信图片_20231127153219](https://github.com/opencv/cvat/assets/31345352/29f78b30-be1f-4b24-96ed-64cc498e225c)
Through the official source code deployment can be automatically annotated, but modified into their own model nuclio function deployment is no problem, but CVAT reported an error, the official function can not be found, there are eight big men have encountered this problem
function.yaml:
metadata:
  name: person_ZDZG-yolov5
  namespace: cvat
  annotations:
    name: person_ZDZG
    type: detector
    framework: pytorch
    spec: |
      [
        { "id": 0, "name": "person" },
      ]

spec:
  description: YOLO v5 via pytorch hub
  runtime: 'python:3.6'
  handler: main:handler
  eventTimeout: 30s
  build:
    image: cvat/ultralytics-yolov5
    baseImage: ultralytics/yolov5:latest-cpu

    directives:
      preCopy:
        - kind: USER
          value: root
        - kind: RUN
          value: apt update && apt install --no-install-recommends -y libglib2.0-0
        - kind: WORKDIR
          value: /opt/nuclio

  triggers:
    myHttpTrigger:
      maxWorkers: 2
      kind: 'http'
      workerAvailabilityTimeoutMilliseconds: 10000
      attributes:
        maxRequestBodySize: 33554432 # 32MB
        port: 32002
  platform:
    attributes:
      restartPolicy:
        name: always
        maximumRetryCount: 3
      mountMode: volume

main.py:
import json
import base64
from PIL import Image
import io
import torch

def init_context(context):
    context.logger.info("Init context...  0%")

    # Read the DL model
    # model = torch.hub.load('ultralytics/yolov5', 'yolov5s')  # or yolov5m, yolov5l, yolov5x, custom
    model = torch.hub.load('/root/yolov5-5.0', 'custom', '/root/yolov5-5.0/weights/person20231123s/weights/best.pt', source='local')
    context.user_data.model = model

    context.logger.info("Init context...100%")

def handler(context, event):
    context.logger.info("Run yolo-v5 model")
    data = event.body
    buf = io.BytesIO(base64.b64decode(data["image"]))
    threshold = float(data.get("threshold", 0.5))
    context.user_data.model.conf = threshold
    image = Image.open(buf)
    yolo_results_json = context.user_data.model(image).pandas().xyxy[0].to_dict(orient='records')

    encoded_results = []
    for result in yolo_results_json:
        encoded_results.append({
            'confidence': result['confidence'],
            'label': result['name'],
            'points': [
                result['xmin'],
                result['ymin'],
                result['xmax'],
                result['ymax']
            ],
            'type': 'rectangle'
        })

    return context.Response(body=json.dumps(encoded_results), headers={},
        content_type='application/json', status_code=200)

**Comment by bsekachev** [comment]
Hello, 

``docker logs nuclio-nuclio-<func_name>`` will help you investigate the problem.
Unfortunately we can't help with custom functions.

**Comment by chang50961471** [comment]
> 你好
> 
> `docker logs nuclio-nuclio-<func_name>`将帮助您调查问题。不幸的是，我们无法帮助自定义函数。

I found that after nuclio function is deployed, sometimes it takes a long time for CVAT to be called. Have you ever encountered this situation
